.circleci/config.yml

```diff
@@ -1014,6 +1020,7 @@ jobs:
       - image: <<pipeline.parameters.docker-image>>
     steps:
       - checkout
+      - skip-if-webui-only
```

Comment:
https://app.circleci.com/pipelines/github/determined-ai/determined/29369/workflows/dc146461-fe7c-4028-8fec-7e17a90b15dd 
this might not work for upstream branches.